Presentation.Web/app/components/local-config/localOptionList/localOptionList.directive.ts

```diff
@@ -131,17 +134,20 @@
                                 operator: "contains"
                             }
                         }
-                    },
-                    {
-                        name: "editOption",
-                        text: "Redigér",
-                        template: "<button type='button' class='btn btn-link' title='Redigér type' ng-click='ctrl.editOption($event)'><i class='fa fa-pencil' aria-hidden='true'></i></button>",
-                        title: " ",
-                        width: 176
                     } as any
                 ]
             };
 
+            if (!this.disableEdit) {
```

Comment:
I think we should also remove the column for "Beskrivelse" - not just the button